moveit_ros/perception/semantic_world/CMakeLists.txt

```diff
@@ -15,4 +15,6 @@ ament_target_dependencies(
   Eigen3
   Boost)
 
+target_link_libraries(moveit_semantic_world PRIVATE ${OpenCV_LIBRARIES})
```

Comment:
Can't we just add `OpenCV` to `ament_target_dependencies` ? See https://github.com/RoboStack/ros-kilted/blob/main/patch/ros-kilted-moveit-ros-perception.osx.patch .